packages/payload/src/database/types.ts

```diff
@@ -481,6 +482,26 @@ export type CreateArgs = {
   select?: SelectType
 }
 
+export type FindDistinctArgs = {
+  collection: CollectionSlug
+  field: string
+  limit?: number
+  locale?: string
+  page?: number
+  req?: Partial<PayloadRequest>
+  sortOrder?: 'asc' | 'desc'
+  where?: Where
+}
+
+export type PaginatedDistinctDocs = {
```

Comment:
Should this be more closely aligned with `PaginatedDocs`? For example, it's missing many of the properties like `nextPage`, `totalPages`, etc. that would make it possible to paginate through distinct results.